keanu-python/tests/test_stats.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pytest
+
+from keanu.vertex import Uniform
+from keanu.algorithm import sample
+from keanu import Model, stats
+
+
+def test_can_get_autocorrelation():
```

Comment:
please add a type annotation `-> None` to each test. This will cause MyPy to pick them up.

You will then have some new MyPy errors which Joanne or I can help you with - you'll need to add some more methods to the numpy stub.

Reply:
Oh I see you've already added `hstack` to the stubs so I guess you know how to do it!

Reply:
Sorry, I had added them and forgot to push. Should be there now.